django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -6,16 +6,60 @@
 from typing import Any, Dict
 
 import requests
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
 from django.core.management.base import BaseCommand, CommandError
 from jwcrypto.jwk import JWK
 from jwcrypto.jws import InvalidJWSSignature
 from jwcrypto.jwt import JWT
+from OpenSSL import crypto
 
 from django_fido.constants import PEM_CERT_TEMPLATE
 from django_fido.models import AuthenticatorMetadata
 from django_fido.settings import SETTINGS
 
 
+class InvalidCert(Exception):
+    """Raised when certificate validation fails."""
+
+
+def verify_certificate(jwt: JWT) -> JWK:
+    """Get (and verify) the signing key from JWT."""
+    # First element in the header is our actual key
+    try:
+        decoding_key = JWK.from_pem(PEM_CERT_TEMPLATE.format(jwt.token.jose_header['x5c'][0]).encode())
+    except ValueError:
+        raise InvalidCert('Cannot decode key.')
+    if SETTINGS.metadata_service['certificate'] is None:
+        return decoding_key
```

Comment:
Do we really want to skip the verification by default?

Reply:
Probably don't have much choice if we are missing the root certificate to validate against.

Reply:
There could be more explicit configuration option to disable verification, because this could lead to omitting the option unintentionally